packages/ai/core/index.ts

```diff
@@ -11,3 +11,4 @@ export * from './registry';
 export * from './tool';
 export * from './types';
 export { cosineSimilarity } from './util/cosine-similarity';
+export { detectImageMimeType } from './util/detect-image-mimetype';
```

Comment:
Torn about exposing the utility. We'd need to document / support it. We could expose it under `/test` or `/internal`?

Reply:
or even copy the utility for now until we can think more about how to best do it. I'm leaning towards including under `ai/test`

Reply:
Will copy it for now.